app/controllers/actors_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ActorsController < ApplicationController
+  def index
+    @actors = Actor.all
```

Comment:
No need for an Actor index! The index action we're looking for is for studios (US 1). Be mindful of not creating actions that you don't need in order to not create more work for yourself!